package.json

```diff
@@ -44,28 +40,30 @@
   "license": "MIT",
   "dependencies": {
     "css-mediaquery": "^0.1.2",
-    "expo-screen-orientation": "~1.1.1"
+    "expo-screen-orientation": "~2.1.0"
```

Comment:
we should make this more permissive - like `expo-screen-orientation@^2.1.0` - so we only need to change it if there is a major version bump with breaking changes